SortApp/SortApp.Presentation/LoadArray/LoadArrayService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using SortApp.ViewModel;
+
+namespace SortApp.Presentation.LoadArray
+{
+	/// <summary>
+	/// Represents the implementation for loadArray service.
+	/// </summary>
+	/// <owner>Oleh Petrenko</owner>
+	public sealed class LoadArrayService : ILoadArrayService
+	{
+		/// <summary>
+		/// Shows window for loading array.
+		/// </summary>
+		/// <owner>Oleh Petrenko</owner>
+		public void ShowLoadArrayWindow(ViewModelBase dataContext)
+		{
+			LoadArrayWindow loadArrayWindow = new LoadArrayWindow
+			{
+				DataContext = dataContext
+			};
+
+			loadArrayWindow.ShowDialog();
```

Comment:
Please, add cancel button to this window (detailed comment is in file below).
Then you should return some value from this method (if user click cancel - ignore selected item).